src/sanctoral/a.yml

```diff
@@ -0,0 +1,555 @@
+#  Sanctoral - A
+
+aaron:
+  canonisationLevel: Saint
+  firstName: Aaron
+  sex: male
+  titles:
+    - Martyr
+
+abachum:
+  canonisationLevel: Saint
+  firstName: Abachum
+  sex: male
+  titles:
+    - Martyr
+  celebrationDate1969: 19/01
+  deathDate: 270
+
+achilleus:
+  canonisationLevel: Saint
+  firstName: Achilleus
+  sex: male
+  titles:
+    - Martyr
+
+adalbert:
+  canonisationLevel: Saint
+  firstName: Adalbert
+  sex: male
+  titles:
+    - Bishop
+    - Martyr
+
+adalbertAndfidelisOfSigmaringen:
+  join:
+    - adalbert
+    - fidelisOfSigmaringen
+
+adalbertOfPrague:
+  canonisationLevel: Saint
+  firstName: Adalbert
+  surname: of Prague
+  sex: male
+  titles:
+    - Bishop
+    - Martyr
+
+adolphKolping:
+  canonisationLevel: Blessed
+  firstName: Adolph
+  surname: Kolping
+  sex: male
+  titles:
+    - Priest
+
+aelredOfRievaulx:
+  canonisationLevel: Saint
+  firstName: Aelred
+  surname: of Rievaulx
+  sex: male
+
+aengus:
+  canonisationLevel: Saint
+  firstName: Aengus
+  sex: male
+  titles:
+    - Bishop
+    - Abbot
+  notes: Oengus
+
+agatha:
+  canonisationLevel: Saint
+  firstName: Agatha
+  sex: female
+  titles:
+    - Virgin
+    - Martyr
+
+agnes:
+  canonisationLevel: Saint
+  firstName: Agnes
+  sex: female
+  titles:
+    - Virgin
+    - Martyr
+
+agnesOfBohemia:
+  canonisationLevel: Saint
+  firstName: Agnes
+  surname: of Bohemia
+  sex: female
+  titles:
+    - Virgin
+
+aidan:
+  canonisationLevel: Saint
+  firstName: Aidan
+  sex: male
+  titles:
+    - Bishop
+
+aidanAndTheSaintsOfLindisfarne:
+  join:
+    - aidan
+    - saintsOfLindisfarne
+
+aidanOfLindisfarne:
+  canonisationLevel: Saint
+  firstName: Aidan
+  surname: of Lindisfarne
+  sex: male
+  titles:
+    - Bishop
+    - Missionary
+
+ailbe:
+  canonisationLevel: Saint
+  firstName: Ailbe
+  sex: male
+  titles:
+    - Bishop
+
+alban:
+  canonisationLevel: Saint
+  firstName: Alban
+  sex: male
+  titles:
+    - Martyr
+
+albanAndJuliusAndAaron:
+  join:
+    - alban
+    - julius
+    - aaron
+
+albertChmielowski:
+  canonisationLevel: Saint
+  firstName: Albert
+  surname: Chmielowski
+  sex: male
+  titles:
+    - Religious
+
+albertinaBerkenbrock:
+  canonisationLevel: Blessed
+  firstName: Albertina
+  surname: Berkenbrock
+  sex: female
+  titles:
+    - Virgin
+    - Martyr
+
+albertoHurtado:
+  canonisationLevel: Saint
+  firstName: Alberto
+  surname: Hurtado
+  sex: male
+  titles:
+    - Priest
+
+albertTheGreat:
+  canonisationLevel: Saint
+  firstName: Albert
+  surname: the Great
+  sex: male
+  titles:
+    - Bishop
+    - Doctor
+    - Abbot
+
+alfonsoRodriguezOlmedo:
+  canonisationLevel: Saint
+  firstName: Alfonso
+  surname: Rodriguez Olmedo
+  sex: male
+  titles:
+    - Priest
+    - Martyr
+
+allSaints:
+  canonisationLevel: Saint
+  firstName: All Saints
+  number: 1000
```

Comment:
In prefer `unknown`, but internally romcal might still treat that has a number (a hight value like here).

Reply:
So how would we resolve this? I don’t have a preference. I’d like to hear something that is better from the _code_ point of view. :smile:

Anyway, I have chosen `1000` because highest number in the current list has 3 digits.

Reply:
And what about `many`?

Reply:
Good idea! :+1: